src/alertSystem/Alert.tsx

```diff
@@ -14,11 +14,15 @@ interface AlertProps {
   prompt?: boolean
   confirm?: boolean
   title?: string
+  okLabel?: number
+  cancelLabel?: number
 }
 interface Options {
   className?: string
   title?: string
   duration?: number
+  okLabel?: number
+  cancelLabel?: number
```

Comment:
Why make this a number? Wouldn't it be better if it was a string so you can pass anything in it? Same goes for `okLabel`

Reply:
I have absolutely no idea why I used that type when I created the PR... It was obviously `string`. This is fixed